src/core/server/stacks/rejectComment.ts

```diff
@@ -48,12 +48,14 @@ const rejectComment = async (
     actionCounts: {},
   });
 
-  // Publish changes to the event publisher.
-  await publishChanges(broker, {
-    ...result,
-    ...counts,
-    moderatorID,
-  });
+  if (broker) {
+    // Publish changes to the event publisher.
+    await publishChanges(broker, {
+      ...result,
+      ...counts,
+      moderatorID,
+    });
+  }
```

Comment:
Should add a `TODO` about why this was added, because the broker is too complex now. Include a reference to CORL-935.